book/http_fundamentals.rst

```diff
@@ -173,7 +173,7 @@ Symfony is architected to match this reality.
     To learn more about the HTTP specification, read the original `HTTP 1.1 RFC`_
     or the `HTTP Bis`_, which is an active effort to clarify the original
     specification. A great tool to check both the request and response headers
```

Comment:
We should now reword the sentence a bit: "Great tools to check [...] while browsing are the [...]"